fdbclient/CMakeLists.txt

```diff
@@ -221,3 +234,8 @@ if(BUILD_AZURE_BACKUP)
   target_link_libraries(fdbclient PRIVATE curl uuid azure-storage-lite)
   target_link_libraries(fdbclient_sampling PRIVATE curl uuid azure-storage-lite)
 endif()
+
+if(BUILD_AWS_BACKUP)
+  target_link_libraries(fdbclient PRIVATE curl awss3sdk_target)
```

Comment:
doesn't `awss3sdk_target` depend on `curl`? Should this be reflected in the aws cmake file?